teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
+    super(name, age, parent_permission)
+    @specialization = specialization
```

Comment:
- [ ] As you know that `super` method sends the parameters to the parent class and in this scenario parent class is Person class and you know that in the Person class constructor age comes before the name parameter 👍 so it would be nice if you make your super method parameters in the same sequence as the parent class has because it will make confusion in the arguments or maybe your app will throw errors 😨

It would look like below ⬇️

```ruby
super(age, name, parent_permission)
```

Reply:
@HAMMAS-SALEEM Done. Thanks